.swiftformat

```diff
@@ -10,7 +10,7 @@
   --elseposition same-line
 
 --enable fileHeader
-  --header "//\n// Copyright 2018-{created.year} Amazon.com,\n// Inc. or its affiliates. All Rights Reserved.\n//\n// SPDX-License-Identifier: Apache-2.0\n//"
+  --header "//\n// Copyright 2018-2021 Amazon.com,\n// Inc. or its affiliates. All Rights Reserved.\n//\n// SPDX-License-Identifier: Apache-2.0\n//"
```

Comment:
This change is to prevent many unrelated file changes. After this PR, or another one in parallel, I will update to remove the date, to match the format of Amplify iOS's https://github.com/aws-amplify/amplify-ios/blob/main/.swiftformat#L13

Reply:
Updating here https://github.com/aws-amplify/aws-appsync-realtime-client-ios/pull/59/files

Reply:
👍